tests/Resources/ResourceResetter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Resources;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * @todo: better name?
+ */
+class ResourceResetter
```

Comment:
It depends if you plan on using it for other resources in the future? (in which case explain its generic purpose in the PHPDoc with potential future use cases)
Or maybe it could be `ResourcesSynchronizer` ?

Reply:
Maybe if you want to keep it more generic you could modify it so `backupImages` would become:

`public static function backupResourceFolder(string $resourceFolder): void` and it would be used via `ResourceResetter::backupResourceFolder(_PS_IMG_DIR_);` then you can create the temp directory based on the string input:

```
    public static function getBackupTestDir(string $resourceFolder): string
    {
        return sys_get_temp_dir() . DIRECTORY_SEPARATOR . md5($resourceFolder);
    }
```

Reply:
I don't want it to be more generic :smile:

Reply:
I got that 😅